Makefile

```diff
@@ -108,19 +108,19 @@ PROTO_GEN      := .gen/proto
 ##### Tools #####
 update-checkers:
 	@printf $(COLOR) "Install/update check tools..."
-	GO111MODULE=off go get -u golang.org/x/tools/cmd/goimports
-	GO111MODULE=off go get -u golang.org/x/lint/golint
-	GO111MODULE=off go get -u honnef.co/go/tools/cmd/staticcheck
-	GO111MODULE=off go get -u github.com/kisielk/errcheck
+	(cd && go get -u golang.org/x/tools/cmd/goimports)
+	(cd && go get -u golang.org/x/lint/golint)
+	(cd && go get -u honnef.co/go/tools/cmd/staticcheck)
+	(cd && go get -u github.com/kisielk/errcheck)
 
 update-mockgen:
 	@printf $(COLOR) "Install/update mockgen tool..."
-	GO111MODULE=off go get -u github.com/golang/mock/mockgen
+	go get -u -t github.com/golang/mock/mockgen
 
 update-proto-plugins:
 	@printf $(COLOR) "Install/update proto plugins..."
-	GO111MODULE=off go get -u github.com/gogo/protobuf/protoc-gen-gogofaster
-	GO111MODULE=off go get -u google.golang.org/grpc
+	(cd && go get -u github.com/temporalio/gogo-protobuf/protoc-gen-gogoslick)
```

Comment:
This is our fork which generates short string enum representation (same as it was before).